server/src/controllers/booking.js

```diff
@@ -87,7 +87,12 @@ const bookingRoom = (req, res, next) => {
           'Other bookings already exist in the requested interval',
           overlapsArr
         );
-      return bookRoom(time, roomId, userId, title, description);
+      else {
+        if (req.body.description === undefined) {
+          return bookRoom(time, roomId, userId, title, 'No Description');
```

Comment:
just make it an empty string `''`
and use the default parameters if possible instead of this undefined if statement